bin/dota/objects/GodScrewdriver.java

```diff
@@ -0,0 +1,47 @@
+package dota.objects;
+
+import dota.logic.GameLogic;
+
+import java.util.ArrayList;
+
+public class GodScrewdriver extends Hero {
+    public GodScrewdriver() {
+        super("Бог-шуруповерт", 70, 80, 10);
+    }
+
+    @Override
+    public void attack(ArrayList<Hero> enemies) {
+        int enemyNumber = GameLogic.chooseEnemy();
```

Comment:
If that static method, import it as static method and use like chooseEnemy(); not like GameLogic.chooseEnemy();